internal/ingest/kafkaingest/serializer/serializer.go

```diff
@@ -17,12 +16,12 @@ type Serializer interface {
 }
 
 type CloudEventsKafkaPayload struct {
-	Id      string    `json:"id"`
-	Type    string    `json:"type"`
-	Source  string    `json:"source"`
-	Subject string    `json:"subject"`
-	Time    time.Time `json:"time"`
-	Data    string    `json:"data"`
+	Id      string `json:"id"`
+	Type    string `json:"type"`
+	Source  string `json:"source"`
+	Subject string `json:"subject"`
+	Time    int64  `json:"time"`
```

Comment:
There is one important consequence of this change: we lose timezone information. It's not necessarily a problem, unless there is a timezone change during a meter aggregation window in which case events may not belong to the right window.

It's as much as of an edge case as it can get, but it's good to know, maybe we should write it down somewhere.